developers/wcs/embeddings/models.md

```diff
@@ -0,0 +1,172 @@
+import GoCode from '!!raw-loader!/\_includes/code/howto/go/docs/model-providers/2-usage-text/main.go';
+import JavaCode from '!!raw-loader!/\_includes/code/howto/java/src/test/java/io/weaviate/docs/model_providers/UsageWeaviateTextEmbeddings.java';
+import JavaCode2 from '!!raw-loader!/\_includes/code/howto/java/src/test/java/io/weaviate/docs/model_providers/UsageWeaviateTextEmbeddingsArcticEmbedLV20.java';
+import JavaImportQueries from '!!raw-loader!/\_includes/code/howto/java/src/test/java/io/weaviate/docs/model_providers/ImportAndQueries.java';
+
+On this page, you can find a list of pre-trained models designed specifically for enterprise retrieval tasks in English and other languages. Additional models and features will be added in the future, so please check back regularly for updates.
+
+## How to choose the right model?
+
+Here are some simple recommendations on when you should use a specific model:
+- **[`Snowflake/snowflake-arctic-embed-m-v1.5`](#snowflake-arctic-embed-m-v1.5)**  
+  Best for datasets that are **primarily in English** with text lengths typically **under 512 tokens**.
+- **[`Snowflake/snowflake-arctic-embed-l-v2.0`](#snowflake-arctic-embed-l-v2.0)**  
+  Ideal for datasets that include **multiple languages** or require **longer context (up to 8192 tokens)**. This model is optimized for robust performance on both English and multilingual retrieval tasks.
+
+Below, you can find a complete list of all available models.
+
+---
+
+## Available models
+
+<!-- TODO[g-despot]: Uncomment section when more models are added
+The following models are available for use with Weaviate Embeddings:
+
+- **[`Snowflake/snowflake-arctic-embed-m-v1.5`](#snowflake-arctic-embed-m-v1.5)**
+- **[`Snowflake/snowflake-arctic-embed-l-v2.0`](#snowflake-arctic-embed-l-v2.0)** (default)
+
+---
+-->
+### `Snowflake/snowflake-arctic-embed-m-v1.5` {#snowflake-arctic-embed-m-v1.5}
+
+- A 109M parameter, 768-dimensional model for enterprise retrieval tasks in English.
+- Trained with Matryoshka Representation Learning to allow vector truncation with minimal loss.
+- Quantization-friendly: Using scalar quantization and 256 dimensions provides 99% of unquantized, full-precision performance.
+- Read more at the [Snowflake blog](https://www.snowflake.com/engineering-blog/arctic-embed-m-v1-5-enterprise-retrieval/), and the Hugging Face [model card](https://huggingface.co/Snowflake/snowflake-arctic-embed-m-v1.5)
+- Allowable `dimensions`: 768 (default), 256
+
+:::info Input truncation
+Currently, input exceeding the model's context windows is truncated from the right (i.e. the end of the input).
+:::
+
+#### Parameters
+
+- `model` (optional): The name of the model to use for embedding generation.
+- `dimensions` (optional): The number of dimensions to use for the generated embeddings. Only available for models that support Matryoshka Representation Learning.
```

Comment:
If we are going to list parameters for each model like this wdyt about removing this clause? Since this section is model-specific `Only available for models that support Matryoshka Representation Learning.`